lib/game_statistics.rb

```diff
@@ -47,6 +47,18 @@ def percentage_visitor_wins
     (visitor_wins.to_f * 100 / data_size).round(2)
   end
 
+  def percentage_ties
+    index = 0
+    tie_wins = 0
+    data_size.times do
+      if @games_hash["away_goals"][index] == @games_hash["home_goals"][index]
```

Comment:
Hey, I see that these 3 percentage methods are very similar! 

Maybe we can get together and discuss how to refactor all of these.

Reply:
Love your thinking. I believe we can throw all of these into a overarching method to call. Would the passes argument be the symbol for type, like :home, :visitor, :tie?